corehq/apps/dhis2/dbaccessors.py

```diff
@@ -1,7 +1,7 @@
 from corehq.util.quickcache import quickcache
 
 
-@quickcache(['domain_name'])
+@quickcache(['domain_name'], timeout=10)
```

Comment:
I had a bug where after saving connection details or dataset maps, the user would still get the cached version. Considering that usually those functions are going to be called daily, I reckoned that if the timeout is longer than the call that needs the cache but less than the amount of time it takes the user to change and save the details, then it would resolve the bug, and still be no less effective when it comes to sending data to DHIS2 24 hours later.

I saw the same pattern being used in [accounting.models](https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/accounting/models.py#L732) and [reports.util](https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/reports/util.py#L479).

Is there a better way to do this?